Security/src/Test-ProxyLogon.ps1

```diff
@@ -207,7 +210,7 @@ begin {
 
                     Get-ChildItem -Path $env:ProgramData -Recurse -ErrorAction SilentlyContinue |
                         ForEach-Object {
-                            If ( $_.Extension -in $zipFilter ) {
+                            If ( $zipFilter -contains $_.Extension ) {
```

Comment:
This works equally well, but won't prevent some issues with PSv2:
PSCustomObject in the line below will likely only return hashtables, which in turn will cause interesting reports (but can be addressed on the caller side)

Reply:
RE: pipeline.  I presume you mean all the trappings of an advanced function.  I'm happy to do that, but I share your concern that it may be overkill for little ROI.

RE: 2.0, That's a great point, I was myopically responding to the specific complaint in #260.  While I'm not fond of the `New-Object` syntax maybe we should move to it.

However, 2.0 is so old at this point I'm not even sure it'll accept the hash as an arg to -Property.  I'd have to install the 2.0 engine just to test, but per other MS comment's on #260 I'm not sure how far we want to go to accommodate 2.0. Especially considering the `[PSCustomObject]@{...}` is leveraged elsewhere.